Build broken on master because of bundle actions command

#666 was merged earlier, but because the PR that removed the signing was merged before, this PR still referenced some methods that got changed, thus breaking the build:

```
$ make build
go build  -ldflags '-w -s -X github.com/deislabs/duffle/pkg/version.Version=0.1.0-ralpha.5+englishrose-44-gae6a7ca' -o /Users/radu/go/src/github.com/deislabs/duffle/bin/duffle github.com/deislabs/duffle/cmd/...
# github.com/deislabs/duffle/cmd/duffle
cmd/duffle/bundle_actions.go:45:42: too many arguments in call to resolveBundleFilePath
        have (string, string, bool, bool)
        want (string, string, bool)
cmd/duffle/bundle_actions.go:50:24: too many arguments in call to loadBundle
        have (string, bool)
        want (string)
make: *** [build] Error 2
```

--- a/cmd/duffle/bundle_actions.go
+++ b/cmd/duffle/bundle_actions.go
@@ -16,7 +16,6 @@ type bundleActionsCmd struct {
 	out          io.Writer
 	home         home.Home
 	bundle       string
-	insecure     bool
 	bundleIsFile bool
 }
 
@@ -36,18 +35,17 @@ func newBundleActionsCmd(w io.Writer) *cobra.Command {
 	}
 
 	cmd.Flags().BoolVarP(&a.bundleIsFile, "bundle-is-file", "f", false, "Indicates that the bundle source is a file path")
-	cmd.Flags().BoolVarP(&a.insecure, "insecure", "k", false, "Do not verify the bundle (INSECURE)")
 
 	return cmd
 }
 
 func (a *bundleActionsCmd) run() error {
-	bundleFile, err := resolveBundleFilePath(a.bundle, a.home.String(), a.bundleIsFile, a.insecure)
+	bundleFile, err := resolveBundleFilePath(a.bundle, a.home.String(), a.bundleIsFile)
 	if err != nil {
 		return err
 	}
 
-	bun, err := loadBundle(bundleFile, a.insecure)
+	bun, err := loadBundle(bundleFile)
 	if err != nil {
 		return err
 	}

--- a/cmd/duffle/bundle_actions_test.go
+++ b/cmd/duffle/bundle_actions_test.go
@@ -12,7 +12,6 @@ func TestBundleActions(t *testing.T) {
 		out:          out,
 		bundleIsFile: true,
 		bundle:       "./testdata/testbundle/actions-test-bundle.json",
-		insecure:     true,
 	}
 
 	if err := cmd.run(); err != nil {

